src/Iesi.Collections/Iesi.Collections.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard1.3;net40;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard1.3;netstandard2.0;net40;net461</TargetFrameworks>
     <Company>NHibernate community</Company>
     <Summary>Additional implementations of System.Collections.Generic.ISet&lt;T&gt;.</Summary>
     <Description>The System.Collections namespace in the .NET Framework provides a number of collection types that are extremely useful for manipulating data in memory. However, some specialized implementations of ISet are not available. Iesi.Collections 4.0 for .Net 4.0 contains the LinkedHashSet (preserves insertion order), the ReadOnlySet and the SynchronizedSet. The latter two wrap an actual set.</Description>
-    <Version>4.0.4</Version>
-    <FileVersion>4.0.4.0</FileVersion>
+    <Version>4.1.0</Version>
+    <FileVersion>4.1.0.0</FileVersion>
     <AssemblyVersion>4.0.0.4000</AssemblyVersion>
```

Comment:
If we follow NHibernate handling, this one should be changed too for a new minor. (Or NHibernate versionning could be changed in 6.0 for putting in `AssemblyVersion` only the major, since with SemVer no binary breaking changes are supposed to happen in a minor.)